helm_deploy/hmpps-tier-to-delius-update/templates/_envs.tpl

```diff
@@ -31,6 +31,9 @@ env:
   - name: COMMUNITY_ENDPOINT_URL
     value: "{{ .Values.env.COMMUNITY_ENDPOINT_URL }}"
 
+  - name: HMPPS_TIER_ENDPOINT_URL
```

Comment:
One of the decisions in dev refinement of https://dsdmoj.atlassian.net/browse/MAS-355 was to get this URL from the message

Reply:
Ugh do we need it for the oauth web client setup though?

Reply:
exactly, so if we want to use HATEOAS we have to construct the webclient on the fly for each call. I like it in principal and I try to get projects to use it but we either go down the rabbit hole or don't try IMO, in this instance we own both services

Reply:
https://spring.io/blog/2020/04/22/spring-hateoas-brings-you-new-ways-to-configure-clients
is this any use?